### DISCUSSION TITLE
How do I check the currently installed version of the webui?

### DISCUSSION THREAD

**Discussion by CalculonPrime** [oobabooga/text-generation-webui#6553] [Q&A] [answered]
Automatic1111 shows the current version in the corner of the UI.  I don't see that with this software.

**Comment by ebaker-github** [comment, marked as answer]
This is the only way I have found to do this.

In a terminal navigate to the text-generation-webui-main folder

Use the Git command window to query the commit hash using the command "git rev-parse HEAD"
C:\PortableApps\text-generation-webui-main>git rev-parse HEAD
cc8c7ed2093cbc747e7032420eae14b5b3c30311

NOTE: If installed on Windows, Install the Git application for Windows and run "git-cmd.exe" from the Git application folder.

Enter the commit hash in the URL:
https: //github.com/oobabooga/text-generation-webui/commit/cc8c7ed2093cbc747e7032420eae14b5b3c30311
(I added a space after https: so the entire URL is displayed here)

Look for the current "version" information in the upper left corner. In this example, the version is 1.16. Click the link the version page will open and provide you with the version date.

**Reply by ethanbeauc** [reply]
I'm pretty sure it does, but does this mean that the portable version of textgen (extracted from the tar.gz) is entirely devoid of version tags anywhere? Kind of annoying when trying to figure out what's changed since I last installed/updated lol.

**Comment by myytautowork-ux** [comment]
The way it brings people [mod apk giveaway](https://nandla.link/) together, whether online or offline, is what makes it special. I was wondering if you've noticed any impact on community engagement when using mod apk for games. It seems like it can really boost player interaction.

**Comment by bloggingxyz9-hash** [comment]
You can usually check the installed version from the release files, Git commit, or package metadata if it's not shown in the UI. We ran into the same thing while working on [website](https://riyadhprayer.com/).
